service/pom.xml

```diff
@@ -85,11 +85,24 @@
         </dependency>
 
         <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>1.4.192</version>
```

Comment:
We should have the version be a property.  Especially because it is used here and on line 180 for the flyway plugin.